content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -135,15 +135,19 @@ Some operating systems have specific configuration requirements:
 - [RedHat Enterprise Linux (RHEL) Requirements](./rhel-requirements.html)
 - [CentOS Requirements](./centos-requirements.html)
 
-For other Linux distributions, check Docker compatibility:
-
-* The instance should run a supported version of Docker engine (1.7.1 or later, minimum 17.06.2-ce, maximum 19.03.11). This also requires a 64-bit distribution with a minimum Linux Kernel version of 3.10.
-    * Replicated 2.32.0 and above required when running Docker 18+.
-    * In Online mode, the installer will install Docker automatically.
-    * In Airgapped mode, Docker should be installed before you begin.
-* For _RedHat Enterprise_ and _Oracle Linux_, you **must** pre-install Docker as these distributions are [not officially supported by Docker Community Edition](https://docs.docker.com/engine/install/#server).
-
-~> **Important:** We do not recommend running Docker under a 2.x kernel.
+For other Linux distributions, check Docker requirements:
+
+- A 64-bit architecture
```

Comment:
This being in the "Other" section makes it read like it's not applicable to CentOS/RHEL. Perhaps we can reorder?

Reply:
Fixed by https://github.com/hashicorp/terraform-website/pull/1980/commits/e03fb3294eee916afe42220e21ba38586b0036db.